compiler/rustc_middle/src/ty/mod.rs

```diff
@@ -23,6 +23,13 @@ pub use adt::*;
 pub use assoc::*;
 pub use generic_args::{GenericArgKind, TermKind, *};
 pub use generics::*;
+// Can't use a glob import here as it would cause
+// ambiguity when importing the actual types implementing
+// the inherent traits from this module.
+// FIXME(clippy#13764): remove this `allow`.
+#[allow(clippy::useless_attribute)]
+#[allow(rustc::non_glob_import_of_type_ir_inherent)]
+use inherent::SliceLike;
```

Comment:
could you just use the qualified path in the RPIT then? easier than having to both silence and then justify it in the import list